packages/relay-compiler/package.json

```diff
@@ -10,7 +10,7 @@
   "homepage": "https://facebook.github.io/relay/",
   "bugs": "https://github.com/facebook/relay/issues",
   "repository": "facebook/relay",
-  "main": "RelayCompilerPublic",
+  "main": "lib/RelayCompilerPublic",
```

Comment:
Hey @Haroenv, thank you for looking into this. It looks like `lib/RelayCompilerPublic` does not exist. This should probably be just `./RelayCompilerPublic`...

Can you squash your changes into single commit?
Thank you!

Reply:
This is exactly what the PR is for. While the file `/RelayCompilerPublic` exists in the source, once published it doesn't, and the main file should be index.js instead of the non-existent file (then) `RelayCompilerPublic`. I'm not sure which tooling depends on the main field, but the other packages had the `lib/xxx` also listed when I checked last time. If possible, the main key can also be removed before publishing? This would also solve the issue I think.

I can squash once we decide on which way to go @alunyov

Reply:
ahh I see what the problem is. @alunyov @kassens do we do CommonJS imports for the relay-compiler package? is it safe to just change the main entry point to point to `lib/*`?